Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -360,12 +360,12 @@ public void EvaluateInvalidTxPublicKey()
         {
             RawTransaction rawTxWithoutSig = new RawTransaction(
                 0,
-                new PrivateKey().PublicKey.ToAddress().ToByteArray(),
-                new byte[][] { },
-                _fx.TxFixture.PublicKey1.Format(false),
+                new PrivateKey().PublicKey.ToAddress().ByteArray,
+                ImmutableArray<ImmutableArray<byte>>.Empty,
+                _fx.TxFixture.PublicKey1.Format(false).ToImmutableArray(),
                 DateTimeOffset.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss.ffffffZ"),
-                new IDictionary<string, object>[0],
```

Comment:
Is this differ with `IImmutableDictionary<string, object>.Empty`?

Reply:
It's an empty *array* of dictionaries, not an empty dictionary.

Reply:
Oh! What about List<ImmutableDictionary<string, object>>.Empty ?

Reply:
As far as I know, there's no `.Empty` static property to `List<T>`.  The reason why `ImmutableDictionary<TKey, TValue>` has `.Empty` static property is it cannot utilize [C#'s collection initializer syntax][1], which requires a data structure to be *mutable*.

[1]: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#collection-initializers

Reply:
Sorry for bothering and delivering effortless comment. I wasn't tried to test 
 `List<ImmutableDictionary<string, object>>.Empty`. It's my mistake.

The main reason why I concern this is I mixed up two things in this PR: `Immutable<T>.Empty` and `T[0]`. And It makes me confused.

What if change this:`new IDictionary<string, object>[0]` to that:`ImmutableList<IImmutableDictionary<string, object>>.Empty` for achieving more stateless? (but I didn't check the performance.)

Reply:
To me that would be fine.  As it's in a unit test, and only a single call, we don't have to think much about performance here.